extension/src/lib/chromeSettings.js

```diff
@@ -34,6 +35,18 @@ fluid.defaults("gpii.chrome.settings", {
         tableOfContentsEnabled: false, // from tableOfContents,
         wordSpace: 1 // from wordSpace
     },
+    model: {
+        settings: "{settings}.options.defaultSettings"
+    },
+    invokers: {
+        updateSettings: {
```

Comment:
This invoker is unnecessary, please invite users to update the model via the ChangeApplier API, perhaps declaratively: https://docs.fluidproject.org/infusion/development/ChangeApplierAPI.html#declarative-style-for-triggering-a-change

Reply:
@amb26 If I remember correctly this is necessary to provide a [default value](https://github.com/jobara/gpii-chrome-extension/blob/GPII-3322/extension/src/lib/chromeSettings.js#L190) for the options to set. It looks like this is necessary for the [web socket connection](https://github.com/jobara/gpii-chrome-extension/blob/GPII-3322/extension/src/lib/wsConnector.js#L78). I didn't write this bit of code, but I believe it's so that the settings can be "reset" by Morphic when a user is keyed out.

Reply:
OK, a doc comment explaining this additional function of the invoker will be helpful